docs-src/content/cli/Transcribe.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "Transcribe Command"
```

Comment:
I might be missing something obvious, but what is the transcribe command, and where is it? I only see 'cubic-cli' as a binary.

Reply:
We are using sub-commands in the cli binary.  This allows us to ship multiple features in a single binary.  You can do things like `cubic-cli transcribe`, `cubic-cli models`, `cubic-cli version`, all from a single binary.  (the aws cli does the same thing.)

How do we need to change the documentation to reflect that?